platform/mellanox/fw.mk

```diff
@@ -5,5 +5,14 @@ MLNX_FW_FILE = fw-SPC-rel-$(subst .,_,$(MLNX_FW_VERSION))-EVB.mfa
 $(MLNX_FW_FILE)_URL = $(MLNX_SDK_BASE_URL)/$(MLNX_FW_FILE)
 SONIC_ONLINE_FILES += $(MLNX_FW_FILE)
 
+MLNX_SN2700_CPLD_ARCHIVE = msn2700_cpld.tar.gz
```

Comment:
>msn2700_cpld.tar.gz [](start = 27, length = 19)

Can we embed the version string into file name, as EVB.mfa?

Reply:
@qiluo-msft inside this archive the version is already embedded into vme file names.